reactive/kotlinx-coroutines-reactive/build.gradle.kts

```diff
@@ -49,3 +49,7 @@ kover {
         }
     }
 }
+
+atomicfu {
+    transformJs = false
+}
```

Comment:
But `kotlinx-coroutines-reactive` doesn't build any JS artifacts. Of the subprojects listed, only `-core`, `-test`, and `test-utils` do. Is this really needed?